docs/source/technical_tutorials/access/aws_access.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>This is ok, but we should consider using rio-cogeo which is already included in MAAP. Will need to check the docs to figure out how to get rio cogeo info inside of python without having to shell out.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>